### DISCUSSION TITLE
blog/notetaking-app-choice

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#29] [Announcements]
# blog/notetaking-app-choice

这篇文章依照作者对记笔记这件事的本质的理解，逐个点评各类新型笔记软件，并且给出选择它们的建议。当然，文章的重点不在于详细介绍和对比各类软件的特性。

https://pathos.page/blog/notetaking-app-choice

**Comment by imjeff** [comment]
经历Onenote、印象笔记等软件数据迁移后，再也不想尝试非本地纯文本保存的笔记软件。日记应用 Day One 也考虑逐步弃用。对比当前主流的几款双链笔记应用，确实 Obsidian PC端略胜一筹，App 却逊色一些。Obsidian 不建议使用太多影响笔记内容的插件，减少依赖。

**Comment by yunyit** [comment]
还没有尝试过 Workflowy，不过最近开始用 Obsidian 作工作笔记管理，能用公司的 Google Drive 备份 vault 分别在公司电脑和个人电脑编辑内容，对于数据安全还是安心一点（起码现在还没有被公司 IT 敲打哈哈

**Reply by Pathsis** [reply]
如果不是依赖大纲编辑器，我觉得坚持 Obsidian 就很好很好。

**Comment by lightstang** [comment]
磨刀磨着磨着玩起来了，柴还一根没砍...

**Comment by 3417157769** [comment]
思源笔记就可以本地化、免费使用

**Comment by rebron1900** [comment]
logseq和Roam有同样的问题，插件对笔记的侵入同样严重。
而且因为无序列表形式的结构，这些笔记在其他md软件中格式会十分奇怪。
所以在没有更好的笔记软件之前，我还是倾向于更推荐Obsidian

**Reply by Pathsis** [reply]
这俩本来就是大纲笔记，与 markdown 结合后，格式就是 `- 文字` 这样的。我觉得倒不是很大的问题。日常写作可以导出为普通 markdown 格式。当然，如果选择了 Obsidian，自然不用再纠结，用就对了。

**Comment by astro-jingtao** [comment]
好奇【大纲编辑器】是类似 Workflowy 或者幕布的那种吗？就我个人来讲 Obsidian 的大纲体验似乎不比这两个差？是 Roam Research 有额外功能，还是我没有解锁正确用法。

**Reply by Pathsis** [reply]
那或许说明了，您并不依赖大纲编辑器。

**Comment by seyeeL** [comment]
请问为什么

> 新型笔记软件之间不可能做到无损迁移。为了尽可能保证较好的可迁移性，尽可能选择纯文本形式保存的笔记软件，以确保将来的笔记能够被正常读取和搜索。
> 还有一个问题值得考虑，那就是利用 AI 技术的可能性。

这两个问题一起的话没有选择obsidian而是选择了tana？因为博主对大纲需求的占比比较大吗

**Reply by Pathsis** [reply]
1.为什么“没有选择obsidian而是选择了tana”？因为我提的那个考虑是我自己的，而我后面提到转到 Tana 是王树义老师的选择，这个事实被我用来说明人们对 Roam Research 的逃离。
2.至于这两个笔记软件选择的考虑，文中已经说明了啊。

**Comment by seyeeL** [comment]
看到最后

> 我自己的最终方案或许是这样的：日常使用 Workflowy ，并且定期备份至 Obsidian，将这两种软件结合着使用。将来如果实践方案成熟，我还会再写一篇文章。

我就是两个一起用的哈哈哈，已经稳定一两年了。

顺便留一下自己的博客~ 
[Seyee's Blog](https://seyee.cc/ )

**Reply by seyeeL** [reply]
重读这篇文章，才发现最后脚注里的5，是我的文章哈哈哈。上次没注意有没有。

**Reply by Pathsis** [reply]
哈哈，确实是你上次评论之后我补充的

**Comment by silascript** [comment]
obsidian是最自由的，不被软件本身捆绑，因为它存储还是markdown格式，其他笔记软件都乱搞，硬是要弄一个自己的格式，而且格式都无法通过普通的文本编辑器直接编辑，对于用户而言，如何存储是一个黑箱，完全没有掌控。而且备份迁移也是obsidian最好的，因为存储就是markdown格式，所以整个流程是透明化！
